src/components/TasksGrid/TasksGrid.jsx

```diff
@@ -0,0 +1,67 @@
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import formatDate from '../../data/formatDate';
+import classes from './TasksGrid.module.css';
+
+const TaskRow = ({ number, progressId, task, startDate, deadlineDate }) => {
+  return (
+    <div className={classes.Row}>
+      <div className={classes.Item}>{number}</div>
+      <Link className={classes.Item} to={`/progress/${progressId}`}>
+        {task}
+      </Link>
+      <div className={classes.Item}>{formatDate(startDate)}</div>
+      <div className={classes.Item}>{formatDate(deadlineDate)}</div>
+    </div>
+  );
+};
+
+const TasksHeaderRow = () => {
+  return (
+    <div className={classes.Row}>
+      <div className={classes.Item}>#</div>
+      <div className={classes.Item}>Name</div>
+      <div className={classes.Item}>Start</div>
+      <div className={classes.Item}>Deadline</div>
+    </div>
+  );
+};
+
+const TasksGrid = ({ data }) => {
+  let rowNumber = 0;
+  const rows = data.map(({ id, /* userId, */ task, start, deadline, taskProgressId }) => {
+    rowNumber += 1;
+
+    return (
+      <TaskRow
+        number={rowNumber}
+        progressId={taskProgressId}
+        key={`${start}${id}`}
+        task={task}
+        startDate={start}
+        deadlineDate={deadline}
+      />
+    );
+  });
+
+  return (
+    <div className={classes.Grid}>
+      <TasksHeaderRow />
+      {rows}
+    </div>
+  );
+};
```

Comment:
It can be reusable component

Reply:
Now it looks like this.
```suggestion
  return (
    <div className={classes.grid}>
      {headerRow}
      {rows}
    </div>
  );
```